.pre-commit-config.yaml

```diff
@@ -68,7 +68,7 @@ repos:
     rev: v3.3.1
     hooks:
     -   id: pyupgrade
-        files: ^(telegram|examples|tests)/.*\.py$
+        files: ^(telegram|examples|tests|docs)/.*\.py$
         args:
           - --py37-plus
```

Comment:
our docs should be python 3.9+ compatible. So the args passed here is wrong?

Reply:
3.7 code is still compatible with 3.9 though - just not voce versa

Reply:
ah right